stories/assets/js/parallax_lines.js

```diff
@@ -0,0 +1,25 @@
+import './viewport';
+import { breakpoints } from './global-variables';
+
+export function parallaxlines() {
```

Comment:
@pallavikadam-srijan Kindly add appropriate Js comments for functions and statements.